lib/glimesh_web/live/user_live/components/tag_selector.ex

```diff
@@ -0,0 +1,38 @@
+defmodule GlimeshWeb.UserLive.Components.TagSelector do
+  use GlimeshWeb, :live_component
+
+  @impl true
+  def render(assigns) do
+    ~L"""
+    <div id="tagify" phx-update="replace">
+      <%= text_input @form, @field,
+        value: @existing_tags,
+        class: "tagify",
+        "data-category": @current_category_id,
+        "data-tags": @tags,
+        "phx-hook": "TagSelector",
+        placeholder: "describe your stream" %>
```

Comment:
Should this be gettext'd? I'm running at like 2% brain atm.

Reply:
Yeah, will add